packages/halo/src/react/index.js

```diff
@@ -0,0 +1,73 @@
+import * as glamor from 'glamor'
+import polyfillFocusWithin from 'focus-within'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import css, { BASE_CLASSNAME } from '../css'
+import * as vars from '../vars'
+
+import withDefaultTheme from './with-default-theme'
+
+polyfillFocusWithin(document)
```

Comment:
there are other ways to do this. i went with this package/approach because it was the most widely used